.travis.yml

```diff
@@ -199,11 +199,44 @@ jobs:
         - mkdir bin ; ln -s /usr/bin/gcc-5 bin/gcc
       install:
         - ccache -z
-        - ccache --max-size=1G
-        - cmake -H. -Bbuild '-DCMAKE_BUILD_TYPE=Release'  '-DCMAKE_CXX_COMPILER=/usr/bin/g++-5'
+        - ccache --max-size=2G
+        - cmake -H. -Bbuild '-DCMAKE_BUILD_TYPE=RelWithDebInfo'  '-DCMAKE_CXX_COMPILER=/usr/bin/g++-5'
```

Comment:
Have these two changes accidentally escaped here from other work?

Reply:
No, they are intentional; the commit message tries to call them out, but it might not be sufficiently clear?

Reply:
I can confirm that this was an intentional change that @tautschnig and I 
discussed in an earlier part of this PR.